### PR TITLE
Move repo for cert-manager-webhook-example

### PR DESCRIPTION
**Pull request by jakexks**
For context: https://github.com/cert-manager/webhook-example/pull/23

### CODE REVIEW

File: config/jobs/cert-manager-webhook-example/cert-manager-webhook-example-presubmits.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 presubmits:
-  jetstack/cert-manager-webhook-example:
```
Comment (munnerz) [inline config/jobs/cert-manager-webhook-example/cert-manager-webhook-example-presubmits.yaml:2]: Can we move this to be a subdirectory of https://github.com/jetstack/testing/tree/master/config/jobs/cert-manager instead? We could then also look to collate cert-manager ones into its own subdirectory there too

Comment (jakexks) [reply config/jobs/cert-manager-webhook-example/cert-manager-webhook-example-presubmits.yaml:2]: That seems sane

**Review by simplyzee — APPROVED** [review]

**Pull request merged**